cobalt/media/base/sbplayer_pipeline.cc

```diff
@@ -1233,9 +1233,11 @@ void SbPlayerPipeline::OnPlayerError(SbPlayerError error,
     case kSbPlayerErrorDecode:
       CallErrorCB(::media::PIPELINE_ERROR_DECODE, message);
       break;
+#if SB_API_VERSION < 16
```

Comment:
How about removing PLAYBACK_CAPABILITY_CHANGED but still keep kSbPlayerErrorCapabilityChanged? We can report PIPELINE_ERROR_DECODE with a specified error message here for kSbPlayerErrorCapabilityChanged. So, under starboard, we can still use kSbPlayerErrorCapabilityChanged.

It's not a recommendation, more like a brainstorm of how we handle the difference between cobalt and chrome. Also @xiaomings.

Reply:
I think it's a good suggestion, so we can get closer to Chrome at the Cobalt level without making a Starboard change.